utils/exporters/blender/addons/io_three/__init__.py

```diff
@@ -257,216 +257,6 @@ def draw(self, context):
         row = layout.row()
         row.prop(obj, 'THREE_export', text='Export')
 
-def get_settings_fullpath():
-    """
-
-    :returns: Full path to the settings file (temp directory)
-
-    """
-    return os.path.join(bpy.app.tempdir, SETTINGS_FILE_EXPORT)
-
-
-def save_settings_export(properties):
-    """Save the current export settings to disk.
-
-    :param properties:
-    :returns: settings
-    :rtype: dict
-
-    """
-    settings = {
-        constants.VERTICES: properties.option_vertices,
-        constants.FACES: properties.option_faces,
-        constants.NORMALS: properties.option_normals,
-        constants.SKINNING: properties.option_skinning,
-        constants.BONES: properties.option_bones,
-        constants.GEOMETRY_TYPE: properties.option_geometry_type,
-
-        constants.MATERIALS: properties.option_materials,
-        constants.UVS: properties.option_uv_coords,
-        constants.FACE_MATERIALS: properties.option_face_materials,
-        constants.MAPS: properties.option_maps,
-        constants.COLORS: properties.option_colors,
-        constants.MIX_COLORS: properties.option_mix_colors,
-
-        constants.SCALE: properties.option_scale,
-        constants.ENABLE_PRECISION: properties.option_round_off,
-        constants.PRECISION: properties.option_round_value,
-        constants.LOGGING: properties.option_logging,
-        constants.COMPRESSION: properties.option_compression,
-        constants.INDENT: properties.option_indent,
-        constants.COPY_TEXTURES: properties.option_copy_textures,
-        constants.TEXTURE_FOLDER: properties.option_texture_folder,
-
-        constants.SCENE: properties.option_export_scene,
-        #constants.EMBED_GEOMETRY: properties.option_embed_geometry,
-        constants.EMBED_ANIMATION: properties.option_embed_animation,
-        constants.LIGHTS: properties.option_lights,
-        constants.CAMERAS: properties.option_cameras,
-
-        constants.MORPH_TARGETS: properties.option_animation_morph,
-        constants.ANIMATION: properties.option_animation_skeletal,
-        constants.FRAME_STEP: properties.option_frame_step,
-        constants.FRAME_INDEX_AS_TIME: properties.option_frame_index_as_time,
-        constants.INFLUENCES_PER_VERTEX: properties.option_influences
-    }
```

Comment:
Yeah, check this branch out and let me know if this works. It only edits the __init__.py and doesn't touch rest of the modules and doesn't break the test cases.
https://github.com/repsac/three.js/tree/bl_options

Apologize in advance. You had to edit a lot of files for this PR.